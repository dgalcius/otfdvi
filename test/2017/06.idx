\indexentry {quantum mechanics|hyperpage}{1}
\indexentry {Bohr, N.|hyperpage}{1}
\indexentry {Born, M.|hyperpage}{1}
\indexentry {de Broglie, L.|hyperpage}{1}
\indexentry {Einstein, A.|hyperpage}{1}
\indexentry {Fermi, E.|hyperpage}{1}
\indexentry {Fock, V.\,A.|hyperpage}{1}
\indexentry {Heisenberg, W.|hyperpage}{1}
\indexentry {Pauli, W.|hyperpage}{1}
\indexentry {Planck, M.|hyperpage}{1}
\indexentry {Schr\"{o}dinger, E.|hyperpage}{1}
\indexentry {Wigner, E.\,P.|hyperpage}{1}
\indexentry {Dirac, P.\,A.\,M.|hyperpage}{1}
\indexentry {Hilbert, D.|hyperpage}{1}
\indexentry {quantum logic|hyperpage}{2}
\indexentry {Aristotle|hyperpage}{2}
\indexentry {Boole, G.|hyperpage}{2}
\indexentry {Foulis, D.\,J.|hyperpage}{2}
\indexentry {Birkhoff, G.|hyperpage}{2}
\indexentry {von Neumann, J.|hyperpage}{2}
\indexentry {Mackey, G.\,W.|hyperpage}{2}
\indexentry {Piron, C.|hyperpage}{2}
\indexentry {camera obscura|hyperpage}{3}
\indexentry {pinhole camera|hyperpage}{3}
\indexentry {Newton, I.|hyperpage}{3}
\indexentry {corpuscular theory|hyperpage}{3}
\indexentry {diffraction|hyperpage}{4}
\indexentry {Grimaldi, F.\,M.|hyperpage}{4}
\indexentry {interference|hyperpage}{4}
\indexentry {Young, T.|hyperpage}{4}
\indexentry {Taylor, G.\,I.|hyperpage}{5}
\indexentry {wave theory|hyperpage}{6}
\indexentry {Huygens, C.|hyperpage}{6}
\indexentry {Young, T.|hyperpage}{6}
\indexentry {Fresnel, A.-J.|hyperpage}{6}
\indexentry {Maxwell, J.\,C.|hyperpage}{6}
\indexentry {photoelectric effect|hyperpage}{8}
\indexentry {Einstein, A.|hyperpage}{8}
\indexentry {photon|hyperpage}{8}
\indexentry {Planck constant|hyperpage}{9}
\indexentry {Planck, M.|hyperpage}{9}
\indexentry {Compton scattering|hyperpage}{9}
\indexentry {Compton, A.|hyperpage}{9}
\indexentry {de Broglie, L.|hyperpage}{9}
\indexentry {particle-wave dualism|hyperpage}{9}
\indexentry {Davisson, C.|hyperpage}{9}
\indexentry {Germer, L.|hyperpage}{9}
\indexentry {Heisenberg, W.|hyperpage}{9}
\indexentry {state|hyperpage}{10}
\indexentry {pure classical state|hyperpage}{10}
\indexentry {phase space|hyperpage}{10}
\indexentry {trajectory|hyperpage}{10}
\indexentry {proposition|hyperpage}{10}
\indexentry {yes-no experiment|hyperpage}{10}
\indexentry {Geiger, H.|hyperpage}{10}
\indexentry {propositional system|hyperpage}{11}
\indexentry {Boolean logic|hyperpage}{11}
\indexentry {Boole, G.|hyperpage}{11}
\indexentry {meet|hyperpage}{11}
\indexentry {negation|hyperpage}{11}
\indexentry {implication|hyperpage}{11}
\indexentry {logic|hyperpage}{11}
\indexentry {join|hyperpage}{12}
\indexentry {orthocomplement|hyperpage}{12}
\indexentry {maximal proposition|hyperpage}{12}
\indexentry {tautology|hyperpage}{12}
\indexentry {minimal proposition|hyperpage}{12}
\indexentry {absurd proposition|hyperpage}{12}
\indexentry {Aristotle|hyperpage}{13}
\indexentry {orthocomplemented lattice|hyperpage}{13}
\indexentry {distributive law|hyperpage}{14}
\indexentry {classical logic|hyperpage}{14}
\indexentry {Boolean algebra|hyperpage}{14}
\indexentry {postulate|hyperpage}{14}
\indexentry {assertion|hyperpage}{14}
\indexentry {statement|hyperpage}{14}
\indexentry {atomic lattice|hyperpage}{14}
\indexentry {atomic proposition|hyperpage}{14}
\indexentry {disjoint propositions|hyperpage}{15}
\indexentry {experiment|hyperpage}{16}
\indexentry {ensemble|hyperpage}{16}
\indexentry {probability measure|hyperpage}{17}
\indexentry {state|hyperpage}{17}
\indexentry {Kolmogorov axiom|hyperpage}{18}
\indexentry {Kolmogorov, A.\,N.|hyperpage}{18}
\indexentry {statistical mechanics|hyperpage}{18}
\indexentry {mixed state|hyperpage}{18}
\indexentry {probability distribution|hyperpage}{18}
\indexentry {Heisenberg, W.|hyperpage}{20}
\indexentry {Heisenberg microscope|hyperpage}{20}
\indexentry {compatible observables|hyperpage}{20}
\indexentry {partial ordering|hyperpage}{22}
\indexentry {meet|hyperpage}{22}
\indexentry {join|hyperpage}{22}
\indexentry {orthocomplement|hyperpage}{22}
\indexentry {compatible propositions|hyperpage}{22}
\indexentry {orthomodular law|hyperpage}{23}
\indexentry {orthomodular lattices|hyperpage}{23}
\indexentry {quantum logic|hyperpage}{23}
\indexentry {Hilbert, D.|hyperpage}{24}
\indexentry {linear span|hyperpage}{24}
\indexentry {atom|hyperpage}{24}
\indexentry {ray|hyperpage}{24}
\indexentry {Piron's theorem|hyperpage}{25}
\indexentry {Piron, C.|hyperpage}{25}
\indexentry {Euclid|hyperpage}{25}
\indexentry {Frobenius, F.\,G.|hyperpage}{25}
\indexentry {quaternions|hyperpage}{25}
\indexentry {Hilbert, D.|hyperpage}{26}
\indexentry {eigenvalue|hyperpage}{26}
\indexentry {eigensubspace|hyperpage}{26}
\indexentry {spectral projection|hyperpage}{26}
\indexentry {Hermite, C.|hyperpage}{27}
\indexentry {probability measure|hyperpage}{27}
\indexentry {Gleason's theorem|hyperpage}{27}
\indexentry {state|hyperpage}{27}
\indexentry {Gleason, A.|hyperpage}{27}
\indexentry {density operator|hyperpage}{28}
\indexentry {density matrix|hyperpage}{28}
\indexentry {pure quantum state|hyperpage}{28}
\indexentry {eigenvector|hyperpage}{29}
\indexentry {eigenstate|hyperpage}{29}
\indexentry {mixed state|hyperpage}{29}
\indexentry {wave function|hyperpage}{30}
\indexentry {degenerate eigenvalue|hyperpage}{30}
\indexentry {expectation value|hyperpage}{31}
\indexentry {Schr\"odinger equation|hyperpage}{33}
\indexentry {Schr\"{o}dinger's cat|hyperpage}{33}
\indexentry {Schr\"{o}dinger, E.|hyperpage}{33}
\indexentry {orthodox interpretation|hyperpage}{34}
\indexentry {wave function collapse|hyperpage}{34}
\indexentry {Einstein, A.|hyperpage}{35}
\indexentry {Mittelstaedt, P.|hyperpage}{36}
\indexentry {Einstein, A.|hyperpage}{38}
