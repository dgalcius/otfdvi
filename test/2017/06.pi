%Publisher ID=DGBOOK
%Project ID=STEFAN1
%MS=STEFAN101
%Total count=7
\newpi{01f01}{{65.96mm}{35.56mm}{tif}} %DPI=600, depth=8
\newpi[136 354 458 492]{01f02}{{113.59mm}{48.68mm}{eps}}
\newpi{01f03}{{73.79mm}{35.98mm}{tif}} %DPI=600, depth=8
\newpi{01f04}{{45.00mm}{32.13mm}{tif}} %DPI=600, depth=8
\newpi{01f05}{{74.95mm}{24.74mm}{tif}} %DPI=1200, depth=1
\newpi{01f06}{{51.56mm}{51.86mm}{tif}} %DPI=600, depth=8
\newpi{01f07}{{47.12mm}{35.48mm}{tif}} %DPI=600, depth=8
