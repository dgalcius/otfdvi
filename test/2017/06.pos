\relax 
\vtexsavepos@pdf {figure1}{3356424sp}{12271743sp}{3}
\vtexsavepos@pdf {figure2}{5594051sp}{25008258sp}{4}
\vtexsavepos@pdf {figure3}{5594051sp}{13344459sp}{4}
\vtexsavepos@pdf {figure3after}{4475231sp}{20318931sp}{5}
\vtexsavepos@pdf {figure4}{5594051sp}{33834717sp}{10}
\vtexsavepos@pdf {table1}{3356424sp}{25810260sp}{11}
\vtexsavepos@pdf {table2}{5594051sp}{23814517sp}{12}
\vtexsavepos@pdf {figure5}{3356424sp}{25540326sp}{15}
\vtexsavepos@pdf {figure6}{5594051sp}{24832816sp}{16}
\vtexsavepos@pdf {figure7}{3356424sp}{17404093sp}{23}
\vtexsavepos@pdf {table3}{5594051sp}{31079932sp}{24}
