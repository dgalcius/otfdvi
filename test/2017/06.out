\BOOKMARK [0][-]{06.chapter.1:1}{\376\377\0001\000\040\000Q\000u\000a\000n\000t\000u\000m\000\040\000l\000o\000g\000i\000c}{}% 1
\BOOKMARK [1][-]{06.section.1:3}{\376\377\0001\000.\0001\000\040\000W\000h\000y\000\040\000d\000o\000\040\000w\000e\000\040\000n\000e\000e\000d\000\040\000q\000u\000a\000n\000t\000u\000m\000\040\000m\000e\000c\000h\000a\000n\000i\000c\000s\000?}{06.chapter.1:1}% 2
\BOOKMARK [2][-]{06.subsection.1:4}{\376\377\0001\000.\0001\000.\0001\000\040\000C\000o\000r\000p\000u\000s\000c\000u\000l\000a\000r\000\040\000t\000h\000e\000o\000r\000y\000\040\000o\000f\000\040\000l\000i\000g\000h\000t}{06.section.1:3}% 3
\BOOKMARK [2][-]{06.subsection.2:8}{\376\377\0001\000.\0001\000.\0002\000\040\000W\000a\000v\000e\000\040\000t\000h\000e\000o\000r\000y\000\040\000o\000f\000\040\000l\000i\000g\000h\000t}{06.section.1:3}% 4
\BOOKMARK [2][-]{06.subsection.3:10}{\376\377\0001\000.\0001\000.\0003\000\040\000L\000i\000g\000h\000t\000\040\000o\000f\000\040\000l\000o\000w\000\040\000i\000n\000t\000e\000n\000s\000i\000t\000y\000\040\000a\000n\000d\000\040\000o\000t\000h\000e\000r\000\040\000e\000x\000p\000e\000r\000i\000m\000e\000n\000t\000s}{06.section.1:3}% 5
\BOOKMARK [1][-]{06.section.2:13}{\376\377\0001\000.\0002\000\040\000C\000l\000a\000s\000s\000i\000c\000a\000l\000\040\000l\000o\000g\000i\000c}{06.chapter.1:1}% 6
\BOOKMARK [2][-]{06.subsection.1:14}{\376\377\0001\000.\0002\000.\0001\000\040\000P\000h\000a\000s\000e\000\040\000s\000p\000a\000c\000e\000\040\000o\000f\000\040\000o\000n\000e\000\040\000c\000l\000a\000s\000s\000i\000c\000a\000l\000\040\000p\000a\000r\000t\000i\000c\000l\000e}{06.section.2:13}% 7
\BOOKMARK [2][-]{06.subsection.2:17}{\376\377\0001\000.\0002\000.\0002\000\040\000P\000r\000o\000p\000o\000s\000i\000t\000i\000o\000n\000s\000\040\000i\000n\000\040\000p\000h\000a\000s\000e\000\040\000s\000p\000a\000c\000e}{06.section.2:13}% 8
\BOOKMARK [2][-]{06.subsection.3:19}{\376\377\0001\000.\0002\000.\0003\000\040\000O\000p\000e\000r\000a\000t\000i\000o\000n\000s\000\040\000w\000i\000t\000h\000\040\000p\000r\000o\000p\000o\000s\000i\000t\000i\000o\000n\000s}{06.section.2:13}% 9
\BOOKMARK [2][-]{06.subsection.4:26}{\376\377\0001\000.\0002\000.\0004\000\040\000A\000x\000i\000o\000m\000s\000\040\000o\000f\000\040\000l\000o\000g\000i\000c}{06.section.2:13}% 10
\BOOKMARK [2][-]{06.subsection.5:29}{\376\377\0001\000.\0002\000.\0005\000\040\000P\000h\000a\000s\000e\000\040\000s\000p\000a\000c\000e\000\040\000f\000r\000o\000m\000\040\000a\000x\000i\000o\000m\000s\000\040\000o\000f\000\040\000c\000l\000a\000s\000s\000i\000c\000a\000l\000\040\000l\000o\000g\000i\000c}{06.section.2:13}% 11
\BOOKMARK [2][-]{06.subsection.6:33}{\376\377\0001\000.\0002\000.\0006\000\040\000C\000l\000a\000s\000s\000i\000c\000a\000l\000\040\000o\000b\000s\000e\000r\000v\000a\000b\000l\000e\000s}{06.section.2:13}% 12
\BOOKMARK [1][-]{06.section.3:38}{\376\377\0001\000.\0003\000\040\000M\000e\000a\000s\000u\000r\000e\000m\000e\000n\000t\000s\000\040\000a\000n\000d\000\040\000p\000r\000o\000b\000a\000b\000i\000l\000i\000t\000i\000e\000s}{06.chapter.1:1}% 13
\BOOKMARK [2][-]{06.subsection.1:39}{\376\377\0001\000.\0003\000.\0001\000\040\000E\000n\000s\000e\000m\000b\000l\000e\000s\000\040\000a\000n\000d\000\040\000m\000e\000a\000s\000u\000r\000e\000m\000e\000n\000t\000s}{06.section.3:38}% 14
\BOOKMARK [2][-]{06.subsection.2:42}{\376\377\0001\000.\0003\000.\0002\000\040\000S\000t\000a\000t\000e\000s\000\040\000a\000s\000\040\000p\000r\000o\000b\000a\000b\000i\000l\000i\000t\000y\000\040\000m\000e\000a\000s\000u\000r\000e\000s}{06.section.3:38}% 15
\BOOKMARK [2][-]{06.subsection.3:46}{\376\377\0001\000.\0003\000.\0003\000\040\000P\000r\000o\000b\000a\000b\000i\000l\000i\000t\000y\000\040\000d\000i\000s\000t\000r\000i\000b\000u\000t\000i\000o\000n\000s\000\040\000a\000n\000d\000\040\000s\000t\000a\000t\000i\000s\000t\000i\000c\000a\000l\000\040\000m\000e\000c\000h\000a\000n\000i\000c\000s}{06.section.3:38}% 16
\BOOKMARK [1][-]{06.section.4:49}{\376\377\0001\000.\0004\000\040\000L\000o\000g\000i\000c\000\040\000o\000f\000\040\000q\000u\000a\000n\000t\000u\000m\000\040\000m\000e\000c\000h\000a\000n\000i\000c\000s}{06.chapter.1:1}% 17
\BOOKMARK [2][-]{06.subsection.1:52}{\376\377\0001\000.\0004\000.\0001\000\040\000P\000a\000r\000t\000i\000a\000l\000\040\000d\000e\000t\000e\000r\000m\000i\000n\000i\000s\000m\000\040\000o\000f\000\040\000q\000u\000a\000n\000t\000u\000m\000\040\000m\000e\000c\000h\000a\000n\000i\000c\000s}{06.section.4:49}% 18
\BOOKMARK [2][-]{06.subsection.2:58}{\376\377\0001\000.\0004\000.\0002\000\040\000A\000x\000i\000o\000m\000s\000\040\000o\000f\000\040\000q\000u\000a\000n\000t\000u\000m\000\040\000l\000o\000g\000i\000c\000\040\000f\000r\000o\000m\000\040\000p\000r\000o\000b\000a\000b\000i\000l\000i\000t\000y\000\040\000m\000e\000a\000s\000u\000r\000e\000s}{06.section.4:49}% 19
\BOOKMARK [2][-]{06.subsection.3:63}{\376\377\0001\000.\0004\000.\0003\000\040\000C\000o\000m\000p\000a\000t\000i\000b\000i\000l\000i\000t\000y\000\040\000o\000f\000\040\000p\000r\000o\000p\000o\000s\000i\000t\000i\000o\000n\000s}{06.section.4:49}% 20
\BOOKMARK [2][-]{06.subsection.4:66}{\376\377\0001\000.\0004\000.\0004\000\040\000L\000o\000g\000i\000c\000\040\000o\000f\000\040\000q\000u\000a\000n\000t\000u\000m\000\040\000m\000e\000c\000h\000a\000n\000i\000c\000s}{06.section.4:49}% 21
\BOOKMARK [2][-]{06.subsection.5:72}{\376\377\0001\000.\0004\000.\0005\000\040\000Q\000u\000a\000n\000t\000u\000m\000\040\000l\000o\000g\000i\000c\000\040\000a\000n\000d\000\040\000H\000i\000l\000b\000e\000r\000t\000\040\000s\000p\000a\000c\000e}{06.section.4:49}% 22
\BOOKMARK [2][-]{06.subsection.6:75}{\376\377\0001\000.\0004\000.\0006\000\040\000P\000i\000r\000o\000n\000'\000s\000\040\000t\000h\000e\000o\000r\000e\000m}{06.section.4:49}% 23
\BOOKMARK [2][-]{06.subsection.7:78}{\376\377\0001\000.\0004\000.\0007\000\040\000S\000h\000o\000u\000l\000d\000\040\000w\000e\000\040\000a\000b\000a\000n\000d\000o\000n\000\040\000c\000l\000a\000s\000s\000i\000c\000a\000l\000\040\000l\000o\000g\000i\000c\000?}{06.section.4:49}% 24
\BOOKMARK [1][-]{06.section.5:81}{\376\377\0001\000.\0005\000\040\000P\000h\000y\000s\000i\000c\000s\000\040\000i\000n\000\040\000H\000i\000l\000b\000e\000r\000t\000\040\000s\000p\000a\000c\000e}{06.chapter.1:1}% 25
\BOOKMARK [2][-]{06.subsection.1:82}{\376\377\0001\000.\0005\000.\0001\000\040\000Q\000u\000a\000n\000t\000u\000m\000\040\000o\000b\000s\000e\000r\000v\000a\000b\000l\000e\000s}{06.section.5:81}% 26
\BOOKMARK [2][-]{06.subsection.2:84}{\376\377\0001\000.\0005\000.\0002\000\040\000S\000t\000a\000t\000e\000s}{06.section.5:81}% 27
\BOOKMARK [2][-]{06.subsection.3:100}{\376\377\0001\000.\0005\000.\0003\000\040\000C\000o\000m\000p\000l\000e\000t\000e\000\040\000s\000e\000t\000s\000\040\000o\000f\000\040\000c\000o\000m\000m\000u\000t\000i\000n\000g\000\040\000o\000b\000s\000e\000r\000v\000a\000b\000l\000e\000s}{06.section.5:81}% 28
\BOOKMARK [2][-]{06.subsection.4:102}{\376\377\0001\000.\0005\000.\0004\000\040\000W\000a\000v\000e\000\040\000f\000u\000n\000c\000t\000i\000o\000n\000s}{06.section.5:81}% 29
\BOOKMARK [2][-]{06.subsection.5:105}{\376\377\0001\000.\0005\000.\0005\000\040\000E\000x\000p\000e\000c\000t\000a\000t\000i\000o\000n\000\040\000v\000a\000l\000u\000e\000s}{06.section.5:81}% 30
\BOOKMARK [2][-]{06.subsection.6:110}{\376\377\0001\000.\0005\000.\0006\000\040\000B\000a\000s\000i\000c\000\040\000r\000u\000l\000e\000s\000\040\000o\000f\000\040\000c\000l\000a\000s\000s\000i\000c\000a\000l\000\040\000a\000n\000d\000\040\000q\000u\000a\000n\000t\000u\000m\000\040\000m\000e\000c\000h\000a\000n\000i\000c\000s}{06.section.5:81}% 31
\BOOKMARK [1][-]{06.section.6:111}{\376\377\0001\000.\0006\000\040\000I\000n\000t\000e\000r\000p\000r\000e\000t\000a\000t\000i\000o\000n\000s\000\040\000o\000f\000\040\000q\000u\000a\000n\000t\000u\000m\000\040\000m\000e\000c\000h\000a\000n\000i\000c\000s}{06.chapter.1:1}% 32
\BOOKMARK [2][-]{06.subsection.1:112}{\376\377\0001\000.\0006\000.\0001\000\040\000Q\000u\000a\000n\000t\000u\000m\000\040\000n\000o\000n\000p\000r\000e\000d\000i\000c\000t\000a\000b\000i\000l\000i\000t\000y}{06.section.6:111}% 33
\BOOKMARK [2][-]{06.subsection.2:114}{\376\377\0001\000.\0006\000.\0002\000\040\000C\000o\000l\000l\000a\000p\000s\000e\000\040\000o\000f\000\040\000w\000a\000v\000e\000\040\000f\000u\000n\000c\000t\000i\000o\000n}{06.section.6:111}% 34
\BOOKMARK [2][-]{06.subsection.3:117}{\376\377\0001\000.\0006\000.\0003\000\040\000C\000o\000l\000l\000a\000p\000s\000e\000\040\000o\000f\000\040\000c\000l\000a\000s\000s\000i\000c\000a\000l\000\040\000p\000r\000o\000b\000a\000b\000i\000l\000i\000t\000y\000\040\000d\000i\000s\000t\000r\000i\000b\000u\000t\000i\000o\000n}{06.section.6:111}% 35
\BOOKMARK [2][-]{06.subsection.4:118}{\376\377\0001\000.\0006\000.\0004\000\040\000H\000i\000d\000d\000e\000n\000\040\000v\000a\000r\000i\000a\000b\000l\000e\000s}{06.section.6:111}% 36
\BOOKMARK [2][-]{06.subsection.5:119}{\376\377\0001\000.\0006\000.\0005\000\040\000Q\000u\000a\000n\000t\000u\000m\000-\000l\000o\000g\000i\000c\000a\000l\000\040\000i\000n\000t\000e\000r\000p\000r\000e\000t\000a\000t\000i\000o\000n}{06.section.6:111}% 37
\BOOKMARK [2][-]{06.subsection.6:120}{\376\377\0001\000.\0006\000.\0006\000\040\000Q\000u\000a\000n\000t\000u\000m\000\040\000r\000a\000n\000d\000o\000m\000n\000e\000s\000s\000\040\000a\000n\000d\000\040\000l\000i\000m\000i\000t\000s\000\040\000o\000f\000\040\000k\000n\000o\000w\000l\000e\000d\000g\000e}{06.section.6:111}% 38
